src/dom-parts/TemplateResult.js

```diff
@@ -98,6 +98,10 @@ const getValue = (value) => {
                 case Array.isArray(value):
                     return value.map(getValue).join('');
                 case value instanceof TemplateResult:
+                    // TODO: instanceof is not working when element-js is used in different modules...
```

Comment:
what is the todo here ?

Reply:
Not to use instanceof maybe?! We could go back to setting some $$__magic_varibales on the classes and check for those.
But there could be another problem. This happens when npm can't hoist element-js and the bundler uses two different versions. If we changed the instanceof check to something else - we could end up with different versions of element-js that are not compatible with each other?